pontoon/base/views.py

```diff
@@ -73,19 +74,25 @@ def home(request):
 def locale(request, locale):
     """Locale view."""
     l = get_object_or_404(Locale, code__iexact=locale)
-
-    projects = (
-        Project.objects.available()
-        .select_related('latest_translation')
```

Comment:
Would `prefetch_related` to fetch the ProjectLocale objects, fed with a `ProjectLocale` queryset with the `select_related` on it work instead?